StripePaymentSheet/StripePaymentSheet/Source/Helpers/STPCardScanner.swift

```diff
@@ -458,43 +433,37 @@ class STPCardScanner: NSObject, AVCaptureVideoDataOutputSampleBufferDelegate {
                 params.expYear = NSNumber(
                     value: Int((topExpiration as! NSString).substring(from: 2)) ?? 0)
             }
-            finish(with: params, error: nil)
+            finish(didSucceed: true)
+            DispatchQueue.main.async {
```

Comment:
We're not checking for `isScanning` anymore, would this still work if the `completeScanIfReady()` call happened when `!isScanning`? `cardScanner(didCompleteWith:)` may now be called regardless of isScanning